.github/workflows/build.yml

```diff
@@ -225,6 +226,7 @@ jobs:
           cp ./jupytercad_lab/dist/jupytercad*.whl jupytercad_lab-0.0.1-py3-none-any.whl
           cp ./jupytercad_app/dist/jupytercad*.whl jupytercad_app-0.0.1-py3-none-any.whl
           cp ./jupytercad_lite/dist/jupytercad*.whl jupytercad_lite-0.0.1-py3-none-any.whl
+          pip install jupytercad_core-0.0.1-py3-none-any.whl jupytercad_lab-0.0.1-py3-none-any.whl jupytercad_app-0.0.1-py3-none-any.whl jupytercad_lite-0.0.1-py3-none-any.whl
```

Comment:
I'd prefer we don't do this and fix https://github.com/jupyterlite/xeus/issues/227 instead. Let's live without settings for now in lite.

Reply:
The next release of jupyterlite-xeus (4.0.0a12) should fix the issue https://github.com/jupyterlite/xeus/pull/234